docs/src/surfaces/modal/modal.jsx

```diff
@@ -479,8 +479,6 @@ export default class ModulesModal extends React.Component {
 
         return (
             <Main page="headers">
-                <TitleBar title="Modal" />
```

Comment:
`<TitleBar />` was removed in here but not in line 3 when importing from `react-cm-ui`.

Reply:
Fixed.